Automattic/jetpack
# Add/social plan redirect urls

#### Changes proposed in this Pull Request:

* Added redirect url for block and classic editor.
* Fixed a bug in the classic editor where the upgrade nudge was showing even if you had a paid plan. 

#### Other information:



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Enable sharing limits (see first line of D85993-code)
* Simulate the scenario of being over the sharing limit or use a site where you're over the sharing limit. You can simulate the shares remaining by setting the value of `numberOfSharesRemaining` to 0 in this file ` projects/js-packages/publicize-components/src/components/form/index.js``set_up_sharing_limits` function. For the classic editor,  set `$this->shares_remaining` to 0. 
* Make sure  the Jetpack plugin disabled and that the Jetpack Social plugin is enabled.
* On both the classic editor: You will see the upgrade nudge when you try to share a post. Finish the checkout flow and you should be redirected back to the editor and you should not see the upgrade nudge. 
* Go to payments admin and remove the plan you added. 
* Try creating a new post. You will see the upgrade nudge again. 
* Enable the block editor.  Finish the checkout flow and you should be redirected back to the editor and you should not see the upgrade nudge. 
* Go to payments admin and remove the plan you added.
